src/libs/actions/Policy/Policy.ts

```diff
@@ -837,18 +838,85 @@ function addBillingCardAndRequestPolicyOwnerChange(
         },
     ];
 
-    const params: AddBillingCardAndRequestWorkspaceOwnerChangeParams = {
-        policyID,
-        cardNumber,
-        cardYear,
-        cardMonth,
-        cardCVV,
-        addressName,
-        addressZip,
-        currency,
-    };
+    if (currency === CONST.PAYMENT_CARD_CURRENCY.GBP) {
+        const params: AddPaymentCardParams = {
+            cardNumber,
+            cardYear,
+            cardMonth,
+            cardCVV,
+            addressName,
+            addressZip,
+            currency,
+            isP2PDebitCard: false,
+        };
+        // eslint-disable-next-line rulesdir/no-api-side-effects-method
+        API.makeRequestWithSideEffects(SIDE_EFFECT_REQUEST_COMMANDS.ADD_PAYMENT_CARD_GBP, params).then((response) => {
```

Comment:
It seems we have already implemented this logic (here: https://github.com/Expensify/App/pull/44321). Let's make the code DRY

Reply:
Its a bit different

Reply:
Is it different enough that we can't replace it using `PaymentMethods.addSubscriptionPaymentCard()`?

Reply:
If its not possible to use it then its not possible to use it but if we can make the code simpler and reuse then we should.